crates/sui-graphql-rpc/src/types/query.rs

```diff
@@ -461,14 +461,14 @@ impl Query {
         before: Option<event::Cursor>,
         filter: Option<EventFilter>,
     ) -> Result<Connection<String, Event>> {
-        let Watermark { hi_cp, .. } = *ctx.data()?;
-
         let page = Page::from_params(ctx.data_unchecked(), first, after, last, before)?;
+        let Watermark { hi_cp, lo_tx, .. } = *ctx.data()?;
         Event::paginate(
             ctx.data_unchecked(),
             page,
             filter.unwrap_or_default(),
             hi_cp,
+            lo_tx,
```

Comment:
nit: put `lo_tx`, before `hi_cp` in the parameter order for `paginate` and the destructure for `Watermark`?